maquetado/setup/scss/application.scss

```diff
@@ -0,0 +1,3 @@
+.title {
+  color: red;
```

Comment:
Its ok to use colors this way on this PR, but we will have to create a `colors.scss` file and create constants for colors using hexas